Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -685,25 +687,27 @@ s32 WiiSockMan::DeleteSocket(s32 s)
 
 void WiiSockMan::Update()
 {
-  s32 nfds = 0;
-  fd_set read_fds, write_fds, except_fds;
-  struct timeval t = {0, 0};
-  FD_ZERO(&read_fds);
-  FD_ZERO(&write_fds);
-  FD_ZERO(&except_fds);
+  const int write_event = (POLLOUT | POLLWRBAND);
+  const int read_event = (POLLIN | POLLRDNORM | POLLRDBAND | POLLPRI);
+  const int error_event = (POLLHUP | POLLERR);
```

Comment:
Any reason you don't make those `constexpr`?

Reply:
I forgot this was a thing. Fixed.